lib/openhab/dsl/monkey_patch/items/metadata.rb

```diff
@@ -238,6 +243,28 @@ def update_from_value(value)
               end
             end
 
+            #
+            # Convert the supplied hash to a java.util.HashMap and all keys and values to java.lang.string objects
+            #
+            # @param [Hash] hash to convert
+            # @param [Java::JavaUtil::HashMap] java_map to put converted objects into
+            #
+            # @return [Java::JavaUtil::HashMap] java_map container strings and other Java HashMaps
+            #
+            def to_java_map(hash, java_map = Java::JavaUtil::HashMap.new)
```

Comment:
Should this be made a class method so it can be used from the MetadataItem class as well? I guess that's why the tests are failing.

Reply:
done